### DISCUSSION TITLE
Increase z-index on datepicker

### DISCUSSION THREAD

**Discussion by dextermontero** [themesberg/flowbite#731] [Ideas]
I request to increase the default value of z-20 to the highest value so that can be send to front, it because my scenario i put the date picker on modal, modal have z-50 to main div and z-40 for the backdrop so the date picker send to back

here the Screenshot
![image](https://github.com/themesberg/flowbite/assets/53195352/ee2d0125-d7d3-4774-aebe-a5f1c9100487)

I added z-index on the style manually in inspect element and then date picker send to front of the modal
![image](https://github.com/themesberg/flowbite/assets/53195352/8844eb46-a4ff-4bef-9400-17e110ae04e9)

**Comment by samsonnagamani** [comment]
Same issue here
